test/functional/FunctionalCodeAction.hs

```diff
@@ -56,11 +56,11 @@ renameTests = testGroup "rename suggestions" [
 
             cars <- getAllCodeActions doc
             cmd <- liftIO $ inspectCommand cars ["Replace with", "putStrLn"]
-            let Just (List [Object args]) = cmd ^. L.arguments
-                Object editParams = args HM.! "fallbackWorkspaceEdit"
+            let Just (List [args]) = cmd ^. L.arguments
+                editParams = args ^. key "fallbackWorkspaceEdit" . _Object
             liftIO $ do
-                "changes" `HM.member` editParams @? "Contains changes"
-                not ("documentChanges" `HM.member` editParams) @? "Doesn't contain documentChanges"
+                (editParams & has (ix "changes"))  @? "Contains changes"
+                not (editParams & has (ix "documentChanges")) @? "Doesn't contain documentChanges"
```

Comment:
I guess they are more `key` cases?

Reply:
yeah, we should be consistent, `key` failed for me in the FunctionalCodeAction module so maybe we should switch to `ix` everywhere